pensjon-brevbaker/src/main/kotlin/no/nav/pensjon/brev/maler/fraser/omregning/ufoeretrygd/Ufoeretrygd.kt

```diff
@@ -0,0 +1,231 @@
+                Nynorsk to "Vi har auka uføretrygda di frå ".expr() + virkningsdato + " fordi du fyller 20 år. Du får no utbetalt uføretrygd med rett som ung ufør.",
+            )
+        }
+    }
+
+    enum class Tillegg { FELLESBARN, SAERKULLSBARN, EKTEFELLE, GJENLEVENDE }
+
+
+    /**
+     * TBU1120, TBU1121, TBU1122, TBU1254, TBU1253, TBU1123
+     */
+    data class BeloepPerMaaned(val perMaaned: Kroner, val utbetalteTillegg: Set<Tillegg>)
+
+    val beloep = createPhrase<LangBokmalNynorsk, BeloepPerMaaned> {
+        val perMaaned = argument().select(BeloepPerMaaned::perMaaned).str()
+        val tillegg = argument().select(BeloepPerMaaned::utbetalteTillegg)
+
+        paragraph {
+            showIf(tillegg.isEmpty()) {
+                textExpr(
+                    Bokmal to "Du får ".expr() + perMaaned + " kroner i uføretrygd per måned før skatt.",
+                    Nynorsk to "Du får ".expr() + perMaaned + " kroner i uføretrygd per månad før skatt.",
+                )
+            }.orShowIf(tillegg.exclusivelyContainsAny(Tillegg.SAERKULLSBARN, Tillegg.FELLESBARN)) {
+                textExpr(
+                    Bokmal to "Du får ".expr() + perMaaned + " kroner i uføretrygd og barnetillegg per måned før skatt.",
+                    Nynorsk to "Du får ".expr() + perMaaned + " kroner i uføretrygd og barnetillegg per månad før skatt.",
+                )
+            }.orShowIf(tillegg.containsOnly(Tillegg.GJENLEVENDE)) {
+                textExpr(
+                    Bokmal to "Du får ".expr() + perMaaned + " kroner i uføretrygd og gjenlevendetillegg per måned før skatt.",
+                    Nynorsk to "Du får ".expr() + perMaaned + " kroner i uføretrygd og attlevandetillegg per månad før skatt.",
+                )
+            }.orShowIf(tillegg.containsOnly(Tillegg.EKTEFELLE)) {
+                textExpr(
+                    Bokmal to "Du får ".expr() + perMaaned + " kroner i uføretrygd og ektefelletillegg per måned før skatt.",
+                    Nynorsk to "Du får ".expr() + perMaaned + " kroner i uføretrygd og ektefelletillegg per månad før skatt.",
+                )
+
+                // EKTEFELLE og (SAERKULLSBARN eller FELLESBARN)
+            }.orShowIf(tillegg.exclusivelyContainsRequiredAndAnyOf(setOf(Tillegg.EKTEFELLE), setOf(Tillegg.SAERKULLSBARN, Tillegg.FELLESBARN))) {
+                textExpr(
+                    Bokmal to "Du får ".expr() + perMaaned + " kroner i uføretrygd, barne- og ektefelletillegg per måned før skatt.",
+                    Nynorsk to "Du får ".expr() + perMaaned + " kroner i uføretrygd, barne- og ektefelletillegg per månad før skatt.",
+                )
+
+                // GJENLEVENDE og (SAERKULLSBARN eller FELLESBARN)
+            }.orShowIf(tillegg.exclusivelyContainsRequiredAndAnyOf(setOf(Tillegg.GJENLEVENDE), setOf(Tillegg.SAERKULLSBARN, Tillegg.FELLESBARN))) {
```

Comment:
exclusivelyContainsRequiredAndAnyOf er litt tunglest. 
selv om contains(Tillegg.GJENLEVENDE) and (contains(Tillegg.GJENLEVENDE) or contains(Tillegg.GJENLEVENDE)) (om jeg forstår dette riktig?) er lengre liker jeg det bedre.

Reply:
Problemet er at det du foreslår ikke er det ekvivalent med `exclusivelyContainsRequiredAndAnyOf`. Om jeg skulle gitt et navn til det du foreslår så ville det vært `containsRequiredAndAnyOf`.

Det funksjonen gjør er:
```
tillegg isSubsetOf (GJENLEVENDE, SAERKULLSBARN, FELLESBARN)
&&
tillegg contains (GJENLEVENDE)
&&
(tillegg contains (SAERKULLSBARN) || tillegg contains (FELLESBARN))
```
Altså er kravet, i tillegg til det du skriver, at `tillegg` ikke kan inneholde noe annet enn de tre.